src/librustc_feature/active.rs

```diff
@@ -44,28 +32,43 @@ macro_rules! declare_features {
             pub declared_lib_features: Vec<(Symbol, Span)>,
             $(
                 $(#[doc = $doc])*
-                pub $feature: bool
+                $feature: bool
             ),+
         }
 
         impl Features {
+            /// Is the given feature enabled?
+            ///
+            /// Panics if the symbol doesn't correspond to a declared feature.
+            pub fn on(&self, name: Symbol) -> bool {
```

Comment:
If the field approach is rejected after all in favor of a method, could it be named `enabled` instead of `on`?
"`on`" looks like it's... an event handler of sorts? `onKeyPress` anyone?
The "on/off" association certainly didn't come immediately to me.

Reply:
Heh; I went with `.active(..)` first, then changed to `.enabled(..)` and finally to `.on(..)` to make it shorter. 😂 `.enabled(..)` is fine by me.